@scripts/plugins/slick.autotooltips.js

```diff
@@ -2,32 +2,31 @@
 (function ($) {
     // Register namespace
     $.extend(true, window, {
-        "Slick": {
-            "AutoTooltips": AutoTooltips
-        }
+        Slick: {
+            AutoTooltips: AutoTooltips,
+        },
     });
     /**
      * AutoTooltips plugin to show/hide tooltips when columns are too narrow to fit content.
      * @constructor
-     * @param {boolean} [options.enableForCells=true]        - Enable tooltip for grid cells
-     * @param {boolean} [options.enableForHeaderCells=false] - Enable tooltip for header cells
-     * @param {number}  [options.maxToolTipLength=null]      - The maximum length for a tooltip
+     * @param {boolean} [autoTooltipOptions.enableForCells=true]        - Enable tooltip for grid cells
+     * @param {boolean} [autoTooltipOptions.enableForHeaderCells=false] - Enable tooltip for header cells
+     * @param {number}  [autoTooltipOptions.maxToolTipLength=null]      - The maximum length for a tooltip
      */
-    function AutoTooltips(_a) {
-        var _b = _a.enableForCells, enableForCells = _b === void 0 ? true : _b, _c = _a.enableForHeaderCells, enableForHeaderCells = _c === void 0 ? false : _c, _d = _a.maxToolTipLength, maxToolTipLength = _d === void 0 ? null : _d;
-        var options = { enableForCells: enableForCells, enableForHeaderCells: enableForHeaderCells, maxToolTipLength: maxToolTipLength };
+    function AutoTooltips(autoTooltipOptions) {
```

Comment:
this was breaking one of the examples. I don't know why cypress was not failing... should we add coverage?

Reply:
What exactly broke the cypress tests? Just the reference? But might be good to get the CI to run them as well :)

Reply:
nvm, I think see the changes. Thanks for fixing ;)

Reply:
Thanks, it was the restructuring in the options breaking when not passing an argument to the constructor.

I'll disable auto formatting in the next PRs